src/sys/socket/sockopt.rs

```diff
@@ -406,7 +406,7 @@ sockopt_impl!(
 #[cfg(feature = "net")]
 sockopt_impl!(
     #[cfg_attr(docsrs, doc(cfg(feature = "net")))]
-    /// Set or receive the Type-Of-Service (TOS) field that is
+    /// Set or receivethe Type-Of-Service (TOS) field that is
```

Comment:
Need a blank space here:

```suggestion
    /// Set or receive the Type-Of-Service (TOS) field that is
```

Reply:
Fixed.